exercises/concept/ticket-please/.meta/Exemplar.elm

```diff
@@ -0,0 +1,67 @@
+module TicketPlease exposing (..)
+
+
+type Ticket
+    = Ticket
+        { status : Status
+        , createdBy : ( User, Int )
+        , assignedTo : Maybe User
+        , comments : List ( User, String )
+        }
+
+
+type Status
+    = New
+    | InProgress
+    | Resolved
+    | Closed
+    | Archived
+
+
+type User
+    = User String
+
+
+emptyComment : ( User, String ) -> Bool
+emptyComment ( _, comment ) =
+    String.isEmpty comment
+
+
+numberOfCreatorComments : Ticket -> Int
```

Comment:
I'm not sure this task adds anything that isn't already done in other tasks? So could probably be removed.